netlify/functions/examples_static_samples_json_cart/examples_static_samples_json_cart.js

```diff
@@ -2,7 +2,7 @@ const cartJSON = require('./cart.json');
 
 const handler = async (ev) => {
   const headers = {
-    'Cache-Control': 'public, max-age=604800, stale-while-revalidate=0',
+    'Cache-Control': 'no-cache',
```

Comment:
Thanks for the fix! It looks like the echo endpoint may have the same issue:

https://github.com/ampproject/amp.dev/blob/cfc06c57705e3466a6ff823bfa875ec7a218a3d2/netlify/functions/examples_api_echo/examples_api_echo.js#L10

That endpoint is also used by the Gmail playground.

Reply:
Modified a few more endpoints.